docs/releases/v1.1.0.rst

```diff
@@ -192,10 +192,10 @@
     General
     -------
 
-    -[api] Datatable no longer supports Python 3.6, because Python 3.6 itself
-      has reached its end of life on 2021-12-23 and will no longer be
-      supported. If you are still using Python 3.6, please consider upgrading.
-      [#3376]
+    -[api] Datatable no longer supports Python 3.6 and 3.7, because they
+      have reached their end of life and will no longer be supported.
+      If you are still using these old Python versions, please consider upgrading.
+      [#3376] [#3434]
```

Comment:
I think this should be a separate changelog entry for whatever release is going to be next.

Reply:
Yeah, we can have it as a separate entry, if it helps to add any clarity.